### PR TITLE
Add jci-trawicki bio

### LINKED ISSUE #17: @jci-trawicki needs a bio
I think @jci-trawicki should have a bio in this repository.

It should meet the following specs:
- [ ] Include his name
- [ ] Be located in the "docs" folder
- [ ] Include hometown and occupation

### PR DESCRIPTION
**Pull request by ghost**
Fixes #17

### CODE REVIEW

File: docs/jci-trawicki.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# jci-trawicki
+
+**Name:** Matthew Trawicki
+
+**Occupation:** Software Engineer
+
+**Hometown:** near Milwaukee
```
Comment (ChiefAG) [inline docs/jci-trawicki.md]: address please

Comment (ahmedj89) [inline docs/jci-trawicki.md]: Nice job! :)

File: docs/jci-trawicki.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# jci-trawicki
+
+**Name:** Matthew Trawicki
```
Comment (pdgessler) [inline docs/jci-trawicki.md]: Middle initial? :+1:

Comment (ghost) [inline docs/jci-trawicki.md]: :ok:

File: docs/jci-trawicki.md (hunk 1 above)
Comment (ghost) [inline docs/jci-trawicki.md]: :cake:

Comment (heightriders) [inline docs/jci-trawicki.md]: Which place is near Milwaukee :sparkling_heart:  ?

**Comment by ChiefAG** [comment]
Go for it :+1:

**Pull request merged**